src/Database/V5/Bloodhound/Types.hs

```diff
@@ -600,6 +602,20 @@ data ReplicaBounds = ReplicasBounded Int Int
                    | ReplicasUnbounded
                    deriving (Eq, Read, Show, Generic, Typeable)
 
+data Compression = CompressionDefault | CompressionBest
```

Comment:
I realize these data constructors are representative of the underlying text in the JSON but I'd like some documentation of what compression algorithm they are likely to force in the Lucene index and why you'd want one versus the other.